app/jobs/benchmark_regression_job.rb

```diff
@@ -0,0 +1,20 @@
+class BenchmarkRegressionJob < ActiveJob::Base
+  queue_as :default
+ 
+  def perform(initiator, benchmark_type, benchmark_result_type, category)
+    commit_objects = BenchmarkRun.where( initiator: initiator, benchmark_type: benchmark_type,
+                     benchmark_result_type: benchmark_result_type).limit(1000)
```

Comment:
If another benchmark result is submitted before this job has been executed, the result will be inaccurate.